allocations/migrations/0005_auto_20210414_1610.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you please rename the part after `0005_` to something that is more descriptive? The auto generated names make it hard to tell what exactly the migration is doing.